src/medCore/gui/toolboxes/medRegistrationSelectorToolBox.cpp

```diff
@@ -207,14 +178,13 @@ void medRegistrationSelectorToolBox::changeCurrentToolBox(int index)
     }
 }
 
-
 //! Clears the toolbox.
 void medRegistrationSelectorToolBox::clear(void)
```

Comment:
The comment for ```medSelectorToolBox::clear``` says _clear input data and the current process toolbox_, which is different from this one (which is as I said below, a good example of why comments for virtual functions should not be copied to derived classes).
This function should at least do the same thing as the base version (by calling it directly).